database/seeds/MinecraftServiceTableSeeder.php

```diff
@@ -243,6 +243,41 @@ private function addCoreOptions()
             'startup' => null,
             'script_install' => $script,
         ]);
+
+        $script = <<<'EOF'
+#!/bin/ash
+# Forge Installation Script
+#
+# Server Files: /mnt/server
+apk update
+apk add curl openjdk8
```

Comment:
Yes, but there are openjdk based containers out there that could be used rather than a generic `alpine` container, that way you don't have to wait on installation.

Plus I believe they are stored on the user's system since other core containers rely on them as a base.